.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 - docker
 language: go
 go:
-- "1.12.x"
+- "1.14.x"
```

Comment:
Previous build failed due to 
```
opsgenie/integration_util.go:73:11: undefined: "errors".As
```

That's because `errors.As` got introduced in go 1.13, bumping to 1.14.x